core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/SubjectsResource.java

```diff
@@ -131,6 +130,18 @@ public void lookUpSchemaUnderSubject(
       }
       matchingSchema = schemaRegistry.lookUpSchemaUnderSubjectUsingContexts(
           subject, schema, normalize, lookupDeletedSchema);
+
+      // If first attempt failed with normalize=false, try again with normalize=true
+      if (matchingSchema == null && !normalize) {
+        log.debug("No matching schema found with normalize = false, retrying with normalize = true");
+        try {
```

Comment:
We can remove this try catch. We already have this exception caught in the outer try